src/routes/events/handlers.js

```diff
@@ -0,0 +1,86 @@
+import handleErrors from '../../lib/apiErrorHandler';
+import {
+  createEvent,
+  findEventsByCollaboratorId,
+  findEventById,
+  findEventsByOwnerId,
+  findEventsByPocId,
+  findEventsByRegionId,
+  updateEvent,
+  destroyEvent,
+} from '../../services/event';
+
+const namespace = 'SERVICE:EVENTS';
+
+const logContext = { namespace };
+
+export const getHandler = async (req, res) => {
+  try {
+    let event;
+    const {
+      eventId,
+      regionId,
+      ownerId,
+      pocId,
+      collaboratorId,
+    } = req.params;
+
+    if (eventId) {
+      event = await findEventById(eventId);
+    } else if (regionId) {
+      event = await findEventsByRegionId(regionId);
+    } else if (ownerId) {
+      event = await findEventsByOwnerId(ownerId);
+    } else if (pocId) {
+      event = await findEventsByPocId(pocId);
+    } else if (collaboratorId) {
+      event = await findEventsByCollaboratorId(collaboratorId);
+    }
+
+    if (!event) {
```

Comment:
**Extremely minor** - if we are lacking one of the params, should we send a bad request code instead of a 404? also, we have a lib installed ```httpCodes``` to use in place of the numbers

Reply:
yeah, I think that makes more sense

Reply:
yeah, I think that makes more sense